cypress/integration/login-test.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-undef */
-const EMAIL = 'malaika.aiyar@gmail.com';
-const PASSWORD = 'WER$wer4';
```

Comment:
@malaikaaiyar If you haven't already, since this password is exposed now, you should change it.

Reply:
I have